### PR TITLE
KYP-760: Track product option ID if SKU is not available

### PR DESCRIPTION
**Pull request by nedick007**
Fallback logic to track product option ID if same product SKU is not available.

### CODE REVIEW

**Comment by nedick007** [comment]
@evgeni-spasov-metrilo ready for review

File: app/code/community/Metrilo/Analytics/Model/Observer.php [hunk 1]
```diff
@@ -172,7 +172,7 @@ public function removeFromCart(Varien_Event_Observer $observer)
         $product = $item->getProduct();
 
         $data = array(
-            'id' => $product->getId()
+            'id' => $product->getSku() ? $product->getSku() : $product->getId()
```
Comment (evgeni-spasov-metrilo) [inline app/code/community/Metrilo/Analytics/Model/Observer.php]: Would you be willing to find what is not okay with this line?
[hint](https://github.com/Metrilo/metrilo-magento2-integration/pull/73/files#diff-3239ea2def5b5dfe803295a77f337128R18-R20), [hint](https://github.com/Metrilo/metrilo-magento2-integration/pull/74/files#diff-928e1864f3b453f8240a57bcc0d7501dR73-R74)

File: app/code/community/Metrilo/Analytics/Model/Observer.php [hunk 2]
```diff
@@ -274,6 +274,7 @@ private function _isRejected($event)
      */
     private function _addToCart($productId, $item, $qty)
     {
+//        $product->getId(), $cartProduct, $item->getQty()
```
Comment (evgeni-spasov-metrilo) [inline app/code/community/Metrilo/Analytics/Model/Observer.php]: Why does this little fella exist?

**Review by evgeni-spasov-metrilo — CHANGES_REQUESTED** [review]

File: app/code/community/Metrilo/Analytics/Model/Observer.php (hunk 1 above)
Comment (nedick007) [reply app/code/community/Metrilo/Analytics/Model/Observer.php]: Fixed.

File: app/code/community/Metrilo/Analytics/Model/Observer.php (hunk 2 above)
Comment (nedick007) [reply app/code/community/Metrilo/Analytics/Model/Observer.php]: Removed.

**Comment by nedick007** [comment]
@evgeni-spasov-metrilo ready

**Review by evgeni-spasov-metrilo — APPROVED** [review]

**Pull request merged**